Modules/LearningSequence/classes/LearnerProgress/class.ilLSLP.php

```diff
@@ -18,14 +16,19 @@
  *
  *********************************************************************/
 
+declare(strict_types=1);
+
 /**
  * lp connector
  */
 class ilLSLP extends ilObjectLP
 {
     public static function getDefaultModes(bool $a_lp_active): array
     {
-        return [ilLPObjSettings::LP_MODE_DEACTIVATED];
+        return [
+            ilLPObjSettings::LP_MODE_DEACTIVATED,
+            ilLPObjSettings::LP_MODE_COLLECTION
+        ];
```

Comment:
Hi @nhaagen, this should be something like
```
if ($a_lp_active) {
    return [
        ilLPObjSettings::LP_MODE_DEACTIVATED,
        ilLPObjSettings::LP_MODE_COLLECTION
    ];
}
return [ilLPObjSettings::LP_MODE_DEACTIVATED];
```
Otherwise it would be possible to choose 'collection' as the default mode for LSO when the learning progress is globally deactivated. (Semantically this all makes very little sense, sorry about that...)